whitelist.py

```diff
@@ -0,0 +1,39 @@
+# Because vulture only performs static analysis, this file is used
+# by vulture as a whitelist module and passed as a parameter when
+# running vulture so that it'll know that certain methods are being
+# used.
+
```

Comment:
While it doesn't explicitly have to be named `whitelist.py` or anything like that, from what it seems like, it just needs to be a Python module, which is unfortunate; this seems to be the method proposed by the developer. 

Looking through the vulture source code, it's not actually treating these modules any differently than other modules, and since vulture basically keeps track of functions/variables/etc. that have been used across the project, it looks like all this sort of whitelist module is doing is simply faking usage of variables/methods so that vulture will think that it's been used already and thus remove the errors. As a result, I don't we would be able to store it in some config file, which is unfortunate; the developer of vulture isn't supporting any special ignore/whitelisting feature, since I suppose this method does the job.

Yeah, the use of TOML is definitely gonna make config/package management a bit less complex; it's already enough work for Python.